src/app/components/search.tsx

```diff
@@ -0,0 +1,78 @@
+"use client"
+import { Card } from "../../components/ui/card";
+import Combobox from "./combo-box";
+import DatePicker from "./date-picker";
+import Travellers from "./travellers";
+import { TravelType } from "../models/flight-parameters";
+import { Button } from "../../components/ui/button";
+import { RadioGroup, RadioGroupItem } from "../../components/ui/radio-group";
+import { useContext } from "react";
```

Comment:
This line should go to the top. Please look for rearranging the imports in all the files